src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2

```diff
@@ -101,36 +115,31 @@ https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model.
+## 1. Wrapped Interferogram
 
-The wrapped interferogram is output in both a georeferenced PNG and KMZ format by default, and the full GeoTIFF file can also be optionally included. Because the fringes are on a wrapped 2-pi scale rather than representing continuous phase difference values, a wrapped interferogram tends to be most useful as a color visualization. This product displays deformation in multiples of half of the sensor wavelength, so areas with fringes that are very close together indicate greater amounts of deformation in the line-of-sight direction from the sensor. Each 2-pi cycle is equivalent to about 2.8 cm in line-of-sight displacement (half the wavelength of the Sentinel-1 sensor, which is about 5.6 cm).
+The wrapped interferogram is output in GeoTIFF file. Because the fringes are on a wrapped 2-pi scale rather than representing continuous phase difference values, a wrapped interferogram tends to be most useful as a color visualization. This product displays deformation in multiples of half of the sensor wavelength, so areas with fringes that are very close together indicate greater amounts of deformation in the line-of-sight direction from the sensor. Each 2-pi cycle is equivalent to about 2.8 cm in line-of-sight displacement (half the wavelength of the Sentinel-1 sensor, which is about 5.6 cm).
 
 The wrapped interferogram image is generated from the interferogram phase prior to unwrapping. The PNG and KMZ files are tagged with _color_phase, and are 2048 pixels wide. The optional GeoTIFF is tagged with _wrapped_phase.tif
 
-*The GeoTIFF version of the wrapped interferogram is optional. Select the "Include Wrapped Phase" option in the On Demand HyP3 Processing Options to include it in the product package.*
 
 ----------------
-## 2. Unwrapped Interferogram TODO: Edit as appropriate
+## 2. Unwrapped Interferogram
 
-The unwrapped interferogram uses a reference point to convert the wrapped 2-pi scale into a continuous scale (of multiples of pi), and is the most important and complicated step in InSAR processing. The Minimum Cost Flow (MCF) approach (https://www.gamma-rs.ch/uploads/media/2002-5_TR_Phase_Unwrapping.pdf) was used for phase unwrapping this product, and the reference point was set to the location of the pixel with the highest 9-pixel neighborhood coherence value. The reference point for this interferogram is located at {{ ref_point_coords['lat'] }} degrees latitude and {{ ref_point_coords['lon'] }} degrees longitude in WGS84 geographic coordinates.
+The unwrapped interferogram uses a reference point to convert the wrapped 2-pi scale into a continuous scale (of multiples of pi), and is the most important and complicated step in InSAR processing. The Minimum Cost Flow (MCF) approach (https://www.gamma-rs.ch/uploads/media/2002-5_TR_Phase_Unwrapping.pdf) was used for phase unwrapping this product, and the reference point was set to the location of the pixel with the highest 9-pixel neighborhood coherence value.
```

Comment:
- Are we not indicating the reference point location for these products?
- I thought that ISCE doesn't use MCF 
- Are we using the same approach for setting the reference point that we use for the GAMMA products?

Reply:
We do not indicate the reference point, and I don't know how the ISCE2 determine the reference point. May check the ISCE2 document to find out.